notification-server/src/processor.rs

```diff
@@ -109,6 +108,33 @@ fn map_transaction_to_notification_information(
                 },
             )]
         }
+        AccountTransactionEffects::TokenUpdate { events } => {
+            use concordium_rust_sdk::protocol_level_tokens::TokenEventDetails;
+            events
+                .iter()
+                .filter_map(|token_event| match &token_event.event {
+                    TokenEventDetails::Mint(protocol_level_tokens::TokenSupplyUpdateEvent {
```

Comment:
Should we handle the mint case (other token events) for the future proving the system when creating a PLT? Currently, only the governance account can mint which is not an average user (has a mobile wallet), but in case we change it.

Reply:
Considered it, but it was requested from the wallet team.
I don't mind adding it, but we can always add it later as well :shrug:

Reply:
I'll handle those events as well, good catch.